### ISSUE TITLE
toast.success returns id but does not open toast

### ISSUE THREAD

**Issue by camilobern** [fkhadra/react-toastify#453]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behaviour?**
Calling the following returns a `toastId` but does not open a toast:

```
   const toastId = toast.success(action.toastMessage, {
        position: toast.POSITION.TOP_CENTER,
        autoClose: 2000,
        onOpen: () => window.alert('Opened'),
        onClose: () => window.alert('Closed')
      });
      window.alert(toastId);
      return state;
    }
```
Also, The `onOpen` and `onClose` are not called.
I'm assuming the toast is getting queued but I don't understand why, this is the first time I'm calling `success(...)`.
**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
* react: 16.11.0
* react-toastify: 5.4.1

**Comment by fkhadra** [comment]
Hey @camilobern,

Did you render the toastContainer somewhere in your app ?

**Comment by camilobern** [comment]
@fkhadra I did.

It was a similar issue to https://github.com/fkhadra/react-toastify/issues/404 and https://github.com/fkhadra/react-toastify/issues/402. 

Fixed by updating to 5.5.0.

**Comment by muhammad-sewts** [comment]
router.push({ name: 'ProjectsView' }).then(() => notifySuccess('Project added successfully!'))

It works for me like this (:
